sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -172,7 +191,7 @@ export class EducationProgramController {
       programId,
```

Comment:
This is not from your PR, the method name get can be renamed to meaningful name. It is a controller method, so rename is just on one place.

Reply:
renamed